docs/getting-started/introduction-to-rook.md

```diff
@@ -0,0 +1,137 @@
+
+Overall, Rook's operator streamlines the deployment and management of Ceph storage on Kubernetes, providing automated configuration, monitoring, and adjustment of the storage cluster.
+
+##### Architecture¶
+![Image title](images/Rook High-Level Architecture.png)
+
+#### How to use it
+Rook helps you use [Ceph storage](https://ceph.com/)  with Kubernetes by taking care of all the setup and monitoring tasks. It makes sure everything is running smoothly and adapts to changes in the storage cluster
+
+Is time to hands dirty, Rook is available in two options:
+
+- Helm Chart: [Rook Helm charts](https://rook.io/docs/rook/latest/Helm-Charts/helm-charts/) are preconfigured templates that simplify the deployment and management of Rook within a Kubernetes cluster. Helm is a package manager for Kubernetes that allows users to define, install, and manage applications and services.
+- CRDS: Rook  uses [CRDs (Custom Resource Definition)](https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/) to define and manage storage resources such as pools, object stores, and file systems. The CRDs provide a way to define these storage-related entities and their associated properties, allowing Rook to interact with and manage them in a Kubernetes-native manner.
+
+##### Requisites 
+
+Based on the Rook documents, if you want to start using rook, you need to have at least the following requirements
+
+* Kubernetes Cluster: we know you probably already have a Kubernetes Cluster,  but if you just want to try Rook before jump on, you can use Minikube and that is fine, of course we encourage recommended to use a real Kubernetes Cluster otherwise probably you will not be able to explore the whole capacity of the Rook
+    * Kubernetes version v1.22 at least is required
+* CPU Architecture:
+    * `amd64 / x86_64`
+    * `arm64`
+* To configure the Ceph storage cluster, at least one of these local storage options are required(not required by Minikube):
+    * Raw devices (no partitions or formatted filesystems)
+    * Raw partitions (no formatted filesystem)
+    * LVM Logical Volumes (no formatted filesystem)
+    * Persistent Volumes available from a storage class in block mode
+
+##### Installing Rook using Helm Chart
+The most easy way to install Rook is by using Helm chart, [Helm](https://helm.sh/) is the package manager for Kubernetes 
+it allows to automate the creation, packaging, configuration, and deployment of Kubernetes applications 
+by combining your configuration files into a single reusable package, you can find more details of helm here [Quickstart Guide](https://helm.sh/docs/intro/quickstart/)
+
+1. First we need to add the rook repository to helm
+  ```console
+  $ helm repo add rook-release https://charts.rook.io/release
+  ```
+2. Clone the `values.yaml` file and replace them with the right values
+  ```console
+  $ curl -s https://raw.githubusercontent.com/rook/rook/master/deploy/charts/rook-ceph/values.yaml -o values.yaml
```

Comment:
I don't think it's a good idea to tell people to download all the values file since it could make upgrades hard... I'd suggest that people only change the values they want to change. @galexrt could you please weigh in on this?